src/BenchmarkDotNet/Configs/DefaultConfig.cs

```diff
@@ -68,6 +68,7 @@ public IEnumerable<IValidator> GetValidators()
             yield return GenericBenchmarksValidator.DontFailOnError;
             yield return DeferredExecutionValidator.FailOnError;
             yield return ParamsAllValuesValidator.FailOnError;
+            yield return ParamsValidator.FailOnError;
```

Comment:
we should also add it to the list of mandatory validators:

https://github.com/dotnet/BenchmarkDotNet/blob/ccbaf08b03b38d1d418756a9e528c60c9f4951d1/src/BenchmarkDotNet/Configs/ImmutableConfigBuilder.cs#L17-L31

so it's applied even when users provide an empty config